runtime/phala/src/lib.rs

```diff
@@ -0,0 +1,1214 @@
+pub fn native_version() -> NativeVersion {
+    NativeVersion {
+        runtime_version: VERSION,
+        can_author_with: Default::default(),
+    }
+}
+
+/// The address format for describing accounts.
+pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
+/// Block type as expected by this runtime.
+pub type Block = generic::Block<Header, UncheckedExtrinsic>;
+/// A Block signed with a Justification
+pub type SignedBlock = generic::SignedBlock<Block>;
+/// BlockId type as expected by this runtime.
+pub type BlockId = generic::BlockId<Block>;
+/// The SignedExtension to the basic transaction logic.
+pub type SignedExtra = (
+    frame_system::CheckSpecVersion<Runtime>,
+    frame_system::CheckTxVersion<Runtime>,
+    frame_system::CheckGenesis<Runtime>,
+    frame_system::CheckEra<Runtime>,
+    frame_system::CheckNonce<Runtime>,
+    frame_system::CheckWeight<Runtime>,
+    pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+);
+/// Unchecked extrinsic type as expected by this runtime.
+pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<Address, Call, Signature, SignedExtra>;
+/// Extrinsic type that has already been checked.
+pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, Call, SignedExtra>;
+/// Executive: handles dispatch to the various modules.
+pub type Executive = frame_executive::Executive<
+    Runtime,
+    Block,
+    frame_system::ChainContext<Runtime>,
+    Runtime,
+    AllPallets,
+>;
+
+type EnsureRootOrHalfCouncil = EnsureOneOf<
+    AccountId,
+    EnsureRoot<AccountId>,
+    pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>,
+>;
+
+construct_runtime! {
+    pub enum Runtime where
+        Block = Block,
+        NodeBlock = opaque::Block,
+        UncheckedExtrinsic = UncheckedExtrinsic,
+    {
+        // System support stuff
+        System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
+        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 1,
+        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 2,
+        Utility: pallet_utility::{Pallet, Call, Event} = 3,
+        Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 4,
+        Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 5,
+        Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 6,
+        Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 7,
+
+        // Parachain staff
+        ParachainInfo: pallet_parachain_info::{Pallet, Storage, Config} = 20,
+        ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned} = 21,
+
+        // Monetary stuff
+        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 40,
+        TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 41,
+        Assets: pallet_assets::{Pallet, Call, Storage, Event<T>} = 42,
```

Comment:
If we put Assets at 42, we should also move Assets in thala to the same slot

Reply:
Thala also 42 <https://github.com/AcalaNetwork/Acala/blob/master/runtime/acala/src/lib.rs#L1833>